.goreleaser_evg.yml

```diff
@@ -0,0 +1,47 @@
+before:
```

Comment:
let EVG have its own file to simplify naming startegy and skip things we don't need on evg (brew)